service-api/src/main/java/greencity/client/UserRemoteClient.java

```diff
@@ -65,6 +71,27 @@ public interface UserRemoteClient {
     @GetMapping("/user/checkByUuid")
     boolean checkIfUserExistsByUuid(@RequestParam(UUID) String uuid);
 
+    /**
+     * Gets current user's positions and all possible related authorities to these
```

Comment:
it returns positions for user with `email` not for the current user

Reply:
edited